src/event_loop.rs

```diff
@@ -328,6 +328,26 @@ impl winit_core::event_loop::run_on_demand::EventLoopExtRunOnDemand for EventLoo
     }
 }
 
+#[cfg(android_platform)]
+impl winit_android::EventLoopExtAndroid for EventLoop {
+    fn android_app(&self) -> &winit_android::activity::AndroidApp {
+        &self.event_loop.android_app
+    }
+}
+
+#[cfg(android_platform)]
+impl winit_android::EventLoopBuilderExtAndroid for EventLoopBuilder {
+    fn with_android_app(&mut self, app: winit_android::activity::AndroidApp) -> &mut Self {
+        self.platform_specific.android_app = Some(app);
+        self
+    }
+
+    fn handle_volume_keys(&mut self) -> &mut Self {
+        self.platform_specific.ignore_volume_keys = false;
+        self
+    }
+}
```

Comment:
This requires the fields to become `pub`, is there a way to pass these function calls through to maybe an internal (on `platform_specific`) implementation of these `Ext` traits? Does such a thing even make sense?

Reply:
I think that's a good idea (and some backends already do that IIRC). But I'll punt on it for now, have added it as a todo item in https://github.com/rust-windowing/winit/issues/3433.